controllers/student.js

```diff
@@ -0,0 +1,41 @@
+const bcrypt = require('bcryptjs');
+const BigPromise = require('../middlewares/BigPromise.js');
+const User = require('../models/userSchema.js');
+
+const registerStudent = BigPromise(async (req, res, next) => {
```

Comment:
Why are we creating this custom promise.? Is it needed.? Also we don't need a seperate file for the **registerStudent** function. You can copy this entire code inside the route file for this route. We can discuss it tomorrow.